src/pages/dev/express/intro.mdx

```diff
@@ -0,0 +1,76 @@
+
+## Fees
+Express transactions require an additional [fee](https://docs.axelar.dev/dev/reference/pricing#callcontractwithtokenexpress-gmp-express) alongside existing GMP fees. This extra fee acts as insurance for the Express relayer for assuming the risk of an Express transaction.
+{/* TODO confirm $1 with team */}
+
+### Gas Estimation
+When estimating gas for an interchain transaction the [estimateGasFee](https://docs.axelar.dev/dev/axelarjs-sdk/axelar-query-api#estimategasfee) function is available to provide the estimated cost. The same function can be used to estimate gas costs for an express transaction. However, for express you also need to specify several options to specify an accurate gas estimation.
+For example an express transaction going from Polygon to Avalanche;
+```
+await api.estimateGasFee(
+    "Polygon",
+    "Avalanche",
+    "MATIC",
+    500000,
+    1.1,
+    "0",
+    {
+      showDetailedFees: true,
+      transferAmount: inputTokenAmount, // In terms of symbol, not unit denom, e.g. use 1 for 1 axlUSDC, not 1000000
+      sourceContractAddress: sourceChainAddress,
+      destinationContractAddress: destinationChainAddress
+      tokenSymbol: "axlUSDC", //aUSDC for testnet
+    } 
+)
+```
+
+## Risks
+
+### Funding
+While the Express solution is an intuitive design pattern for allowing for fast interchain transactions it is not without any risks. Namely if anything changes on the source chain from the time the transaction is sent to the time it is finalized (due to a [block reorg](https://cointelegraph.com/explained/what-is-chain-reorganization-in-blockchain-technology) for example) and the transaction on the destination chain has already executed then that could lead to lost funds on the source chain as the transaction will have been instantly executed on the destination chain. Due to the potential risk here, funds will need to be provided to onboard to express. Liquidity must be provided by a party to underwrite the risk.
+
+### Execution
+On the destination chain an Axelar relayer triggers the `_execute()`/ `_executeWithToken()` function to trigger a transaction. The GMP payload being used can be assumed to be true as it passes through the [validation process](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/4d01eb6ea1b5413520d9ecab804f815103021148/contracts/AxelarGateway.sol#L233) on the Axelar Gateway. In an express call however the execution on the destination chain occurs before the source chain finalizes, meaning that you cannot rely on this validation process. To mitigate, data that is being passed in you can make use of the unique [CommandId](https://github.com/axelarnetwork/axelar-core/blob/main/docs/cli/axelard_query_evm_command.md). The `CommandId` is a unique identifier, which is available on the destination chain you're transacting to. With your unique commanId you can reconstruct the message that was passed in 
+{/* TO DO PROVIDE EXAMPLE FOR FIX^ */}
+
+
```

Comment:
We should add a link to the axelar-examples example and explain how to use it? A full code sample would be really helpful. Maybe a diagram of the flows too?

Reply:
Added link to examples, can add diagram too